docs/src/components/FileUploaderAlert.tsx

```diff
@@ -0,0 +1,11 @@
+import { Alert } from '@aws-amplify/ui-react';
+
+export const FileUploaderAlert = () => {
+  return (
+    <Alert variation="info" role="none">
+      StorageManager has been renamed to FileUploader. The FileUploader export
+      is available since version 3.3.0, previous versions must still use
+      StorageManager.
```

Comment:
```suggestion
      Beginning with @aws-amplify/ui-react-storage@v3.3.0, the StorageManager
      component has been renamed to FileUploader. To migrate, upgrade to the
      latest version of @aws-amplify/ui-react-storage and replace StorageManager
      related imports with their corresponding FileUploader equivalents.

      StorageManager will continue to receive official support until the bext
      next major version release of @aws-amplify/ui-react-storage, at which
      point it will be fully replaced by FileUploader.
```